README.md

```diff
@@ -2,87 +2,116 @@
 
 ## Overview
 
-JetBrains IDEA plugin for interacting with Kubernetes and OpenShift clusters.
+JetBrains IDEA plugin that allows to browse, create, modify and delete resources on Kubernetes or OpenShift clusters.
 The plugin provides functionalities and user experiences that are very close to the Kubernetes extension for vscode (https://marketplace.visualstudio.com/items?itemName=ms-kubernetes-tools.vscode-kubernetes-tools).
 
-This plugin is currently in Preview Mode.
+It is available for install from the [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/15921-kubernetes-by-red-hat).
 
-![](images/demo1.gif)
+![editor push scenario](images/editor-push.gif)
 
-### Kubernetes & OpenShift resource tree
-The plugin offers a tool window with a tree that displays all the resources that exist on a Kubernetes or OpenShift cluster.
-The tree displays all the contexts that exist in your kube config (~/.kube/config).
-The tree is dynamically updated when external tools like kubectl, oc, etc. change the kube config. 
+> **NOTE:** This plugin is in Preview mode. The plugin support for Kubernetes or OpenShift clusters is strictly experimental - assumptions may break, commands and behavior may change!
+
+## Key Features
+ - Edit Kubernetes resource manifests and apply them to your cluster
+ - View Kubernetes and OpenShift clusters in an explorer tree view
+ - Tree View supports dynamic update of K8s resources
+ - Allow Push and Pull Feature to keep the editor in sync with the Kubernetes resource
+ - Support Creation and Deletion of new Kubernetes resources on Cluster
+ - Navigate into Kubernetes workloads, services, pods and nodes
+ - Switching contexts and namespaces within Kubernetes
+* Browse Kubernetes and OpenShift cluster resources in dynamically update tree
+* Delete resources on cluster
+* Edit resources on cluster backed by kubernetes resource schema
+* Create new resources on cluster
+* Editor kept in sync with the resource on cluster by pushing & pulling
+
```

Comment:
@adietish There are some duplicate lines being added. We need to remove the ones starting with '*'.

Reply:
@mohitsuman thx for spottig. Looks like merging via github has problems. Correcting those problem by lookup up your comments above.